client/swarm.go

```diff
@@ -0,0 +1,60 @@
+package client
+
+import (
+	"encoding/json"
+	"net/url"
+
+	"github.com/docker/engine-api/types/swarm"
+	"golang.org/x/net/context"
+)
+
+// SwarmInit initializes the Swarm.
+func (cli *Client) SwarmInit(ctx context.Context, req swarm.InitRequest) (string, error) {
```

Comment:
This is probably the *ultimate* nit but, all other methods have their own file, why not swarm ones ? :angel: :stuck_out_tongue_closed_eyes: